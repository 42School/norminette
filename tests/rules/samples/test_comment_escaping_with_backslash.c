/* *\
/

#include <stdio.h>

int main() {
  for (int i=0; i<2;++i)
    printf("%d\n", i);
}

/**/