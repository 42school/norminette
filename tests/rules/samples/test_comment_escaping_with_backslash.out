[36mtest_comment_escaping_with_backslash.c[0m - [32mIsComment[0m In "GlobalScope" from "None" line 1":
		<MULT_COMMENT=/* */> <NEWLINE>
[36mtest_comment_escaping_with_backslash.c[0m - [32mIsEmptyLine[0m In "GlobalScope" from "None" line 3":
		<NEWLINE>
[36mtest_comment_escaping_with_backslash.c[0m - [32mIsPreprocessorStatement[0m In "GlobalScope" from "None" line 4":
		<HASH> <IDENTIFIER=include> <SPACE> <LESS_THAN> <IDENTIFIER=stdio> <DOT> <IDENTIFIER=h> <MORE_THAN> <NEWLINE>
[36mtest_comment_escaping_with_backslash.c[0m - [32mIsEmptyLine[0m In "GlobalScope" from "None" line 5":
		<NEWLINE>
[36mtest_comment_escaping_with_backslash.c[0m - [32mIsFuncDeclaration[0m In "GlobalScope" from "None" line 6":
		<INT> <SPACE> <IDENTIFIER=main> <LPARENTHESIS> <RPARENTHESIS> <SPACE> 
[36mtest_comment_escaping_with_backslash.c[0m - [32mIsBlockStart[0m In "Function" from "GlobalScope" line 6":
		<LBRACE> <NEWLINE>
[36mtest_comment_escaping_with_backslash.c[0m - [32mIsControlStatement[0m In "Function" from "GlobalScope" line 7":
		<SPACE> <SPACE> <FOR> <SPACE> <LPARENTHESIS> <INT> <SPACE> <IDENTIFIER=i> <ASSIGN> <CONSTANT=0> <SEMI_COLON> <SPACE> <IDENTIFIER=i> <LESS_THAN> <CONSTANT=2> <SEMI_COLON> <INC> <IDENTIFIER=i> <RPARENTHESIS> <NEWLINE>
[36mtest_comment_escaping_with_backslash.c[0m - [32mIsFunctionCall[0m In "ControlStructure" from "Function" line 8":
		<SPACE> <SPACE> <SPACE> <SPACE> <IDENTIFIER=printf> <LPARENTHESIS> <STRING="%d\n"> <COMMA> <SPACE> <IDENTIFIER=i> <RPARENTHESIS> <SEMI_COLON> <NEWLINE>
[36mtest_comment_escaping_with_backslash.c[0m - [32mIsBlockEnd[0m In "Function" from "GlobalScope" line 9":
		<RBRACE> <NEWLINE>
[36mtest_comment_escaping_with_backslash.c[0m - [32mIsEmptyLine[0m In "GlobalScope" from "None" line 10":
		<NEWLINE>
[36mtest_comment_escaping_with_backslash.c[0m - [32mIsComment[0m In "GlobalScope" from "None" line 11":
		<MULT_COMMENT=/**/> 
test_comment_escaping_with_backslash.c: Error!
Error: INVALID_HEADER       (line:   3, col:   1):	[97mMissing or invalid 42 header[0m
Error: SPACE_BEFORE_FUNC    (line:   6, col:   4):	[94mspace before function name[0m
Error: NO_ARGS_VOID         (line:   6, col:  10):	[0mEmpty function argument requires void[0m
Error: BRACE_NEWLINE        (line:   6, col:  12):	[93mExpected newline before brace[0m
Error: TOO_FEW_TAB          (line:   7, col:   1):	[0mMissing tabs for indent level[0m
Error: FORBIDDEN_CS         (line:   7, col:   3):	[0mForbidden control structure[0m
Error: SPACE_REPLACE_TAB    (line:   7, col:   3):	[0mFound space when expecting tab[0m
Error: SPC_AFTER_OPERATOR   (line:   7, col:  13):	[94mmissing space after operator[0m
Error: SPC_BFR_OPERATOR     (line:   7, col:  13):	[94mmissing space before operator[0m
Error: SPC_AFTER_OPERATOR   (line:   7, col:  18):	[94mmissing space after operator[0m
Error: SPC_BFR_OPERATOR     (line:   7, col:  18):	[94mmissing space before operator[0m
Error: SPC_AFTER_OPERATOR   (line:   7, col:  20):	[94mmissing space after operator[0m
Error: TOO_FEW_TAB          (line:   8, col:   1):	[0mMissing tabs for indent level[0m
Error: SPACE_REPLACE_TAB    (line:   8, col:   5):	[0mFound space when expecting tab[0m
